README.md

```diff
@@ -44,4 +44,4 @@ Start Infrasim Service
 Configure Infrasim
 -------------------
 
-You can configure your own legacy hardware through **/etc/infrasim/infrasim.conf**.
+You can configure your own legacy hardware through **/etc/infrasim/infrasim.yml**.
```

Comment:
A little bit hard to understand by "own legacy hardware". How about a description by "compose and customize a virtual server with configurations your real server has by adjusting /etc/infra...."